sky/packages/sky/lib/src/services/fetch.dart

```diff
@@ -53,7 +54,7 @@ Future<UrlResponse> fetchUrl(String relativeUrl) async {
   UrlRequest request = new UrlRequest()
     ..url = url
     ..autoFollowRedirects = true;
-  return await fetch(request);
+  return fetch(request);
```

Comment:
If this CL was applied, fetchUrl would work with and without the async modifier. Now that fetchUrl doesn't use await, it is more efficient to drop the async.